cupy/core/raw.pyx

```diff
@@ -51,6 +51,18 @@ cdef class RawKernel:
         kern = _get_raw_kernel(self.code, self.name, self.options)
         kern(grid, block, args, **kwargs)
 
+    @property
+    def attributes(self):
+        """Returns an object containing runtime kernel attributes.
+
+        Returns:
+            attributes (FuncAttributes): A python class containing the
+                kernel's attributes. For example, ``attributes.numRegs``
+                corresponds to the number of registers used by the kernel.
+        """
+        kern = _get_raw_kernel(self.code, self.name, self.options)
+        return cupy.cuda.driver.funcGetAttributes(kern.ptr)
```

Comment:
How about returning attributes as dict, like as done in `cupy.cuda.Device().attributes`?

Reply:
I have updated to construct a dictionary now (within raw.pyx instead of driver.pyx).